django/dispatch/dispatcher.py

```diff
@@ -64,10 +66,6 @@ def connect(self, receiver, sender=None, weak=True, dispatch_uid=None):
 
                 Receivers must be able to accept keyword arguments.
 
-                If receivers have a dispatch_uid attribute, the receiver will
```

Comment:
I don't think so and there is no code in there which try to access dispatch_uid on the receiver ever.